test/metabase/api/task_test.clj

```diff
@@ -9,7 +9,7 @@
    [toucan2.tools.with-temp :as t2.with-temp]))
 
 (def ^:private default-task-history
-  {:id true, :db_id true, :started_at true, :ended_at true, :duration 10, :task_details nil})
+  {:id true, :db_id true, :started_at true, :ended_at true, :duration 10, :task_details nil :status "success"})
```

Comment:
I don't understand how these `true` values work. Perhaps worth a comment?

Reply:
it's an old convention we used before we had the magic `=?` by calling `mt/boolean-ids-and-timestamps` on the results.

Reply:
Thanks for explaining. I was suggesting a comment on this map, but not a blocker. I guess we want to migrate to `=?` eventually? I'd add a backlog item for that and link think to it in the comment to encourage boy scouts.